vignettes/estimate_time_varying_severity.Rmd

```diff
@@ -20,162 +20,88 @@ editor_options:
 knitr::opts_chunk$set(
   collapse = TRUE,
   comment = "#>",
-  fig.align = "left",
-  fig.width = 700 / 80,
-  fig.height = 400 / 80,
-  dpi = 80
+  dpi = 300,
+  fig.width = 5, fig.height = 3
 )
 ```
 
-# Overview
+There are many biological, epidemiological and behavioural reasons why the severity of a disease might change during the course of an outbreak, including the introduction of vaccines or therapeutics, reducing the relative risk of death, or the emergence of pathogen variants, which may alter the risk of hospitalisation or death associated with infection.
 
-This vignette outlines how to use _cfr_ to estimate how the severity of a
-disease changes over the course of an ongoing outbreak. We wish to estimate 
-how common severity quantities change over time: 
-the *case fatality risk (CFR)*, *infection hospitality risk (IFR)* or 
-*hospitalisation fatality risk (HFR)*. 
+Estimating the severity of an outbreak that has undergone substantial changes is of critical importance for surveillance and planning purposes.
```

Comment:
this could be removed as it doesn't really add much

Reply:
Thanks, removed